### DISCUSSION TITLE
Add website link to repo's homepage

### DISCUSSION THREAD

**Discussion by jffaust** [TheHappyKoala/Harmony-of-the-Spheres#173] [General]
Hello,

From the website, we can find the GitHub repo quite easily but it's not the same the other way around. There's a link called **DEMO** at the top of the readme but at first glance I didn't think this was the main site. It would be nice if gravitysimulator.org/ was added to the repo's **About** section.

Here's an example of another repo using this feature: 
![image](https://user-images.githubusercontent.com/4367222/198880154-07fb0b00-7998-4189-9f62-c2280d788bca.png)

Thanks for making this great software :)

**Comment by TheHappyKoala** [comment]
@jffaust Howdy! Apologies for the delayed response - forgot about these discussion threads - but I think that's a great idea. Will take care of it right away, and thank you for your kind words. Have an awesome day!

**Comment by TheHappyKoala** [comment]
@jffaust Howdy! Apologies for the delayed response - forgot about these discussion threads - but I think that's a great idea. Will take care of it right away, and thank you for your kind words. Have an awesome day!
